src/renderer/components/configure-recording-set-page/index.tsx

```diff
@@ -1,112 +1,44 @@
 import log from 'electron-log';
-import React, { FC, Fragment, MouseEventHandler, useContext, useEffect, useState } from 'react';
-import Col from 'react-bootstrap/Col';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import { useTranslation } from 'react-i18next';
-import CSSTransition from 'react-transition-group/CSSTransition';
-import useLocalStorage from 'react-use/lib/useLocalStorage';
+import React, { FC, MouseEventHandler, useState } from 'react';
+import { useLocalStorage, usePrevious } from 'react-use';
 
-import { RecordingProjectContext } from '../../contexts';
-import { PROJECT_CONFIG_FILENAME } from '../../env-and-consts';
-import { Consumer, RecordingProject, RecordingSet, ScaleKey, SupportedOctave } from '../../types';
-import {
-  checkFileExistence,
-  deleteFolder,
-  ensureFolderExists,
-  getLSKey,
-  join,
-  naiveSerialize,
-  readFile,
-  writeFile,
-} from '../../utils';
-import BackButton from '../back-button';
-import NextButton from '../next-button';
-
-import AddRecordingSetButton from './add-recording-set-button';
-import CreatedRecordingSetList from './created-recording-set-list';
-import SetMetaConfiguration from './set-meta-configuration';
-import SetRecordingListConfiguration from './set-recording-list-configuration';
-import { BuiltInRecordingList } from './types';
+import { Consumer, RecordingSet } from '../../types';
+import { checkFileExistence, getLSKey, join } from '../../utils';
 
+import { ConfigureRecordingSet } from './configure-recording-set';
 import './show-details.scss';
-
-interface ProjectFile extends RecordingProject {
-  name: string;
-  recordingSets: RecordingSet[];
-}
-
-const DUMMY_PROJECT_FILE = {
-  name: '',
-  rootPath: '',
-  recordingSets: [],
-};
-
-const BUILT_IN_RECORDING_LISTS: BuiltInRecordingList[] = [
-  'デルタ式英語リストver5 (Delta English Ver. 5)',
-  'Z式CVVC-Normal (Z Chinese CVVC - Normal)',
-];
-
+import './index.scss';
+import { PreviewPage } from './preview-page';
+
+/*
```

Comment:
TS Docs have two `*` on the first line, like so:
```
/**
 * This is a TS Doc.
 */
```

Try typing just `/**` and press Enter and let VSCode generate the TS Doc block for you.

Reply:
Bump